sdcm/utils/adaptive_timeouts/__init__.py

```diff
@@ -1,26 +1,53 @@
 import logging
 import time
+from concurrent.futures import ThreadPoolExecutor
 from contextlib import contextmanager
 from enum import Enum
 from typing import Any
 
-from sdcm.sct_events.system import SoftTimeoutEvent
+from sdcm.sct_events.system import SoftTimeoutEvent, HardTimeoutEvent
 from sdcm.utils.adaptive_timeouts.load_info_store import NodeLoadInfoService, AdaptiveTimeoutStore, ESAdaptiveTimeoutStore, \
     NodeLoadInfoServices
+from sdcm.utils.features import is_tablets_feature_enabled
 
 LOGGER = logging.getLogger(__name__)
 
+TABLETS_SOFT_TIMEOUT = 1 * 60 * 60
```

Comment:
@pehala 

this is the important part, that would start surfacing issues across the board, since it's a "new" requirement for tablets, i.e. the expectation that each of those commands would take less the an hour

Reply:
I am pretty sure bootstrap is fine with this and we could lower it down even more, but for decomission I think we can exceed this easily

@cezarmoise what are decomission times in your incremental scaling algorithm?